tools/lint/main_ni.c

```diff
@@ -895,7 +900,20 @@ main_ni(int argc, char *argv[])
         /* print the list of schemas */
         print_list(c.out, c.ctx, c.data_out_format);
     } else {
-        if (c.schema_out_format) {
+        if (c.feature_param_format) {
+            for (uint32_t u = 0; u < c.schema_modules.count; u++) {
```

Comment:
We usually declare a variable at the beginning of a function. But here it seems not to be followed. Maybe you can also add a refactoring commit and fix it. I'll leave it up to you.